bonsai/Helpers/DateExt.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  DateExt.swift
+//  bonsai
+//
+//  Created by hoang on 12.12.2021.
+//
+
+import Foundation
+
+extension Date {
+    
+    private struct TransactionHeaderTitle {
+        static let today = "Today"
+        static let yesterday = "Yesterday"
+        
+        static func makeAnotherDate(date: Date) -> String {
```

Comment:
I think `var longStyledDescription: String` would be more expressive

Reply:
Get rid of structure, and change to simple method